signing_clients/apps.py

```diff
@@ -138,6 +138,7 @@ def __str__(self):
 
 class Manifest(list):
     version = '1.0'
+    extra_newline = False
```

Comment:
You should add a simple comment explaining the issue and referencing the bug number